build.gradle

```diff
@@ -15,6 +15,12 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 version = '6.0.0-SNAPSHOT'
 
+configurations.all {
```

Comment:
Let's not check in this change.  You can always temporarily enable it if you need.

Reply:
Sorry, didnt notice this was included in the commit. I have removed this now.